start-from-zero/index.css

```diff
@@ -44,27 +94,50 @@ body, html {
 form {
     display: flex;
     flex-direction: column;
-    
     background: white;
     padding-top: 30px;
     padding-bottom: 30px
 }
 
 .blue-line {
     text-align: center;
-    height: 50px;
-    line-height: 50px;
     background: #03A9F4;
+    height: 50px;
+    line-height: 48px;
     color: white;
+    font-size: 16px;
+    -webkit-box-shadow: 0px 3px 15px -2px rgba(0,0,0,0.79);
+    box-shadow: 0px 3px 15px -2px rgba(0,0,0,0.79);
+    position: relative;
+    z-index: 10000;
```

Comment:
I just wanted to put that element on top of the login form. I must admit that I've exaggerated a little with that z-index.